content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -1255,6 +1255,54 @@ const users = await prisma.user.createMany({
 })
 ```
 
+### `createManyAndReturn()`
+
+`createManyAndReturn` creates multiple records and returns the resulting objects.
+
+:::info
+
+`createManyAndReturn()` is only available in Prisma ORM version 5.14.0 and up.
+`createManyAndReturn()` is only available for PostgreSQL, CockroachDB, and SQLite.
+
+:::
+
+#### Options
```

Comment:
This addition changes the anchor "name" of all the "Options" headlines below this (as those are just incrementally numbered). Are there no links to these headlines anywhere in our docs that need to be adapted (plus 1 in the anchor name)? Otherwise these links will not point to the _wrong_ headline.

Reply:
Double checked. We only link to `#options-7` in the docs, which happens before this.